tutorials/ingest-real-time-websocket-data.md

```diff
@@ -163,21 +171,32 @@ Each price event gives you multiple data points about the given trading pair
 such as the name of the exchange, and the current price. You can also
 occasionally see `heartbeat` events in the response; these events signal
 the health of the connection over time.
+At this point the websocket connection is working successfully to pass data.
+    
+</Procedure>
+    
+To ingest the data into your Timescale service, you need to implement the 
+`on_event` function.    
+</Collapsible>
+
+<Collapsible heading="The real-time dataset">
```

Comment:
Collapsibles are automatically L3 headings, which means you have some weirdness going on with heading levels here:

![image](https://github.com/timescale/docs/assets/3914967/cf0a5684-6b3f-4325-95e0-d0207995d4c8)